File: bad128mb.java
import java.io.FileOutputStream;
import java.io.IOException;
import java.util.Random;
 
public class BadDataGenerator {
    public static void main(String[] args) {
        try (FileOutputStream fos = new FileOutputStream("bad128java.dat")) {
            Random random = new Random();
            byte[] buffer = new byte[1024 * 1024]; // 1 MB buffer
 
            for (int i = 0; i < 128; i++) { // Write 128 MB
                random.nextBytes(buffer);
                fos.write(buffer);
            }
            System.out.println("Bad data generated: bad128java.dat");
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: bad128mb.py
import random

# Define the size of the data (1 MB)
size_in_bytes = 1024 * 1024  # 1 MB

# Open the file to write non-securely generated data
with open('bad.dat', 'wb') as f:
    for _ in range(size_in_bytes):
        # Writing one byte at a time using the non-secure random generator
        f.write(random.getrandbits(8).to_bytes(1, 'little'))

print("Generated 1 MB of non-secure random data in bad.dat.")
